agent/resources/resources_linux.go

```diff
@@ -93,6 +113,12 @@ func (c *cgroupWrapper) setupCgroup(task *api.Task) error {
 		return errors.Wrapf(err, "resource: setup cgroup: unable to create cgroup at %s for task: %s", cgroupRoot, task.Arn)
 	}
 
+	// echo 1 > memory.use_hierarchy
+	err = c.ioutil.WriteFile(filepath.Join(c.cgroupPath, memorySubsystem, cgroupRoot, memoryUseHierarchy), []byte(strconv.Itoa(1)), os.FileMode(0))
```

Comment:
this too, please split this into multiple lines

Reply:
how do i keep missing this one. FIXED 😬